diff -r -u rpmbuild.orig/SPECS/shim.spec rpmbuild/SPECS/shim.spec
--- rpmbuild.orig/SPECS/shim.spec	2015-12-24 10:59:57.145009906 -0800
+++ rpmbuild/SPECS/shim.spec	2015-12-24 12:57:51.353394938 -0800
@@ -1,12 +1,12 @@
 Name:           shim
 Version:        0.9
-Release:        {{shim_rel}}
+Release:        {{shim_rel}}.visa
 Summary:        First-stage UEFI bootloader
 
 License:        BSD
 URL:            http://www.codon.org.uk/~mjg59/shim/
 Source0:	https://github.com/mjg59/shim/releases/download/%{version}/shim-%{version}.tar.bz2
-Source1:	centos.crt
+Source1:	visa.crt
 # currently here's what's in our dbx: # nothing.
 #Source2:	dbx.esl
 Source3:	shim-find-debuginfo.sh
