diff -r -u rpmbuild.orig/SPECS/kernel.spec rpmbuild/SPECS/kernel.spec
--- rpmbuild.orig/SPECS/kernel.spec	2015-12-09 03:53:38.000000000 -0800
+++ rpmbuild/SPECS/kernel.spec	2015-12-19 23:07:06.369297451 -0800
@@ -807,9 +807,9 @@
     make %{?cross_opts} -s mrproper
 
     cp %{SOURCE11} .	# x509.genkey
-    cp %{SOURCE12} .	# extra_certificates
-    cp %{SOURCE15} .	# rheldup3.x509
-    cp %{SOURCE16} .	# rhelkpatch1.x509
+#    cp %{SOURCE12} .	# extra_certificates
+#    cp %{SOURCE15} .	# rheldup3.x509
+#    cp %{SOURCE16} .	# rhelkpatch1.x509
 
     cp configs/$Config .config
 
@@ -849,7 +849,8 @@
     fi
 # EFI SecureBoot signing, x86_64-only
 %ifarch x86_64
-    %pesign -s -i $KernelImage -o $KernelImage.signed -a %{SOURCE13} -c %{SOURCE13}
+    pesign -n %{certdb} -c %{certname} -s -i $KernelImage -o $KernelImage.signed
+    #% pesign -s -i $KernelImage -o $KernelImage.signed -a %{SOURCE13} -c %{SOURCE13}
     mv $KernelImage.signed $KernelImage
 %endif
     $CopyKernel $KernelImage $RPM_BUILD_ROOT/%{image_install_path}/$InstallName-$KernelVer
diff -ur rpmbuild.orig/SOURCES/x509.genkey rpmbuild/SOURCES/x509.genkey
--- rpmbuild.orig/SOURCES/x509.genkey	2016-01-16 01:19:47.763639251 -0800
+++ rpmbuild/SOURCES/x509.genkey	2016-01-16 01:20:23.537639251 -0800
@@ -5,9 +5,9 @@
 x509_extensions = myexts

 [ req_distinguished_name ]
-O = CentOS
-CN = CentOS Linux kernel signing key
-emailAddress = security@centos.org
+O = Visa
+CN = Visa Linux embedded kernel signing key
+emailAddress = security@ovn.visa.com

 [ myexts ]
 basicConstraints=critical,CA:FALSE
